### DISCUSSION TITLE
Something wrong about KB5025305

### DISCUSSION THREAD

**Discussion by ZouNuoLan** [valinet/ExplorerPatcher#1344] [Q&A] [answered]
I installed the latest Explorer Patcher on my PC which is in the Release Preview Builds, and have fun for a while. But about 2 month ago (I guess), I can't install the KB5025305 update. I diceded to unintall the EP, following the guidence and uninstalled EP successfully. But iI still can not install this update. So I just let it go until today. I reinstall EP agian and uninstall EP by admin, hope it can solve this update problem. But it failed. Is anyone have the same problem like me? I truly hope to solve this update problem.

**Comment by topia** [comment]
What makes you think that ExplorerPatcher is blocking the installation of Windows updates? I believe it's a very rare situation.

**Reply by ZouNuoLan** [reply]
I hope so, But I searched something about this problem in some website. Someone has the same problem like me. Also it's only a quite few users has the problem. Rare is true.

**Reply by topia** [reply]
Hmm. In your experience, you were able to successfully update Windows immediately after uninstalling ExplorerPatcher, correct? 
I think it might be helpful to spot the problem if you provide related information, such as the logs from a failed update, detailed steps, and so on.

As this rarity, you may have a few unusual settings in ExplorerPatcher or another software causes that failure (in combination with ExplorerPatcher). For the former, I suggest trying "resetting ExplorerPatcher setting and restarting Windows" before applying an update.

**Reply by ZouNuoLan** [reply]
Yes. I installed EP since Jan.3 this year, and I installed every Windows update successfully except the KB5025305. 
About EP, I only changed the appearance from Windows11 Style to Windows10 Style. Then I changed nothing.
 When it failed to install the update, it also give me an error code 0x800f0905. I searched for some page from the search engine about this error code.
One of the reasons is if the system running on the HyperV or a Virtualmachine, it will report this error code while failed to install Windows update. After I uninstalled the EP few weeks ago, my WSA can not boot successfully in someday. It happened while I get the KB5025305 upate, and also It failed to install.

I found install and running WSA on the system maybe caused the local system turn to running on the Virtualmachine status. Perhapts it's the main reason why I can not install the update.

Back to the WSA, I found the WSA can not boot correctly is the Virtual Machine Platform feature is disabled by unknown reason. So I enable it again, reboot this machine. but still failed to boot the WSA. turn to the feature page it shows two Virtualmachine platform features I can enable. One is on and the Others is off. I try to enabled annother Virtualmachine platform feature which is shows disabled for many times, still failed. I turn to Uninstall the WSA and disable all features about WSA. The two Virtualmachine platform features are all disalbe now.
But I still can not install this update.

I have uninstalled EP by admin two days ago, and rebooted the system.Now I check the update again and wait it download and install. Still failed and shows the same error code. 
Maybe the only way to solve this problem is Reinstall the system?

**Reply by topia** [reply]
I don't think your issue was caused by installing ExplorerPatcher; ExplorerPatcher can't cause such problems. At least it can inject into Explorer, the start menu, and many processes which pull shell extensions. No one implements that, but it may crash these processes.
However, critical Windows update processes, such as TrustedInstaller, are not included.
I thought your problem was caused by ExplorerPatcher's additional files, but it shouldn't be applicable to your explained problem.

And sadly, I don't have any advice for you... Some pages said 0x800f0905 is DISM error `CBS_E_NO_ACTIVE_EDITION` (in CBS.log), but on that page, professional (may be paid) supported them.

**Reply by ZouNuoLan** [reply]
I appreciate your suggestion. These words still help me a lot. I will try to find some way to make it back to normal. Thank you！ (●'◡'●)

**Comment by ZouNuoLan** [comment, marked as answer]
I have made my system back to normal successfully.  I used a official Windows image downloaded from MS, and intalled it with save all files. I mean I uesd a system image to "upgrade" my system. While it has done. All of these problem has been resolved.
I will reinstall EP in some day. It's a very useful tool for Win11. Because of its BAD start menu. Tile is better for me. I love it.😊
